tests/test_distributions.py

```diff
@@ -150,6 +157,26 @@ def test_contains():
     assert c._contains(1.5)
     assert not c._contains(3)
 
+    ilu = distributions.IntUniformDistribution(low=2, high=12)
+    assert not ilu._contains(0.9)
+    assert ilu._contains(2)
+    assert ilu._contains(4)
+    assert ilu._contains(6)
+    assert ilu._contains(12)
+    assert not ilu._contains(12.1)
+    assert not ilu._contains(13)
+
+    # IntLogUniformDistribution with a 'q' parameter.
```

Comment:
Maybe a typo:
```suggestion
    # IntLogUniformDistribution with a 'step' parameter.
```

Reply:
Thanks!  Indeed, it is copied from https://github.com/optuna/optuna/blob/2fd7fc693d12ec0ed93af5dce760da2a5671d6dd/tests/test_distributions.py#L136 Should I fix both?

Reply:
Oh, it's great to fix the existing comment too if you can.